packages/atomic/src/components/atomic-folded-result-list/atomic-folded-result-list.tsx

```diff
@@ -107,11 +88,17 @@ export class AtomicFoldedResultList implements InitializableComponent {
    */
   @Prop({reflect: true}) imageSize?: ResultDisplayImageSize;
   /**
-   * @deprecated use `imageSize` instead.
+   * TODO:
    */
-  @Prop({reflect: true}) image: ResultDisplayImageSize = 'icon';
-
-  private renderingFunction?: (result: FoldedResult) => HTMLElement = undefined;
+  @Prop({reflect: true}) public collectionField?: string;
+  /**
+   * TODO:
+   */
+  @Prop({reflect: true}) public parentField?: string;
+  /**
+   * TODO:
+   */
+  @Prop({reflect: true}) public childField?: string;
```

Comment:
Is `display` omitted on purpose? I think grid displays could still benefit from folded results?

Reply:
I recall suggesting that we remove `display`. I think whether we want to include folded results in grid displays or not is probably gonna depend on Johnny's design, so I think adding support for it would be better in a separate PR.